zebra-utils/src/bin/openapi-generator/main.rs

```diff
@@ -8,6 +8,7 @@ use rand::{distributions::Alphanumeric, thread_rng, Rng};
 use serde::Serialize;
```

Comment:
Sent a suggestion PR for this file as github don't allow me to suggest that part.

https://github.com/ZcashFoundation/zebra/pull/9485